training/training.module

```diff
@@ -0,0 +1,479 @@
+    'file' => 'training.pages.inc',
+    'expanded' => true,
+  );
+  $menu['training/theme/table'] = array(
+    'title' => 'Table Theme',
+    'description' => 'This page is Table Theme page',
+    'page callback' => 'training_theme_table',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/theme/image'] = array(
+    'title' => 'Image Theme',
+    'description' => 'This page is Image Theme page',
+    'page callback' => 'training_image_callback',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/theme/access'] = array(
+    'title' => 'Access Theme',
+    'description' => 'This page is Access Theme page',
+    'page callback' => 'training_access_callback',
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+  $menu['training/form'] = array(
+    'title' => 'Form Example',
+    'description' => 'This page is Form Example',
+    'page callback' => 'training_example_form_callback',
+    'page arguments' => array('training_google_form'),
+    'access callback' => TRUE,
+    'file' => 'training.pages.inc',
+  );
+
+  return $menu;
+}
+
+/**
+ * Check access for page Custom Access Menu Item.
+ *
+ * @param string $role_name
+ *   The role required for access.
+ *
+ * @return bool
+ *   True if the acting user has the role specified.
+ */
+function training_custom_access($role_name) {
+  global $user;
+
+  return in_array($role_name, $user->roles);
+}
+
+/**
+ * Displays a specific string, depending on a number in the address bar.
+ *
+ * @param int $arg
+ *   Integer argument in the address bar.
+ *
+ * @return string
+ *   A string that corresponds to an integer in the address bar, or NULL if there is no match.
+ */
+function digit_load($arg) {
+  switch ($arg) {
+    case 1:
+      $arg = 'one';
+      break;
+    case 2:
+      $arg = 'two';
+      break;
+    case 3:
+      $arg = 'three';
+      break;
+    case 99:
+      $arg = 'Almost there';
+      break;
+    default:
+      $arg = NULL;
+  }
+
+  return $arg;
+}
+
+/**
+ * Title callback to rename the title dynamically.
+ *
+ * @return string
+ *   A string that contents dynamic title.
+ */
+function training_menu_title_callback() {
+  global $user;
+  $user_name = isset($user->name) ? $user->name : 'anonymous';
+
+  return 'Dynamic title: username = ' . $user_name;
+}
+
+/**
+ * Implements hook_permission().
+ */
+ function training_permission() {
+  $permission = array(
+    'train' => array(
+      'title' => 'Access protected menu example',
+      'description' => 'Use this to control access to page Permissioned Menu Item of training module',
+    ),
+    'train_theme' => array(
+      'title' => 'Access protected content',
+      'description' => 'Use this to control access to page Access Theme of training module',
+    ),
+  );
+
+  return $permission;
+}
+
+/**
+ * Creates example form.
+ *
+ * @param array $form
+ *   Array of form elements.
+ *
```

Comment:
remove blank line
Same tags aren't separated by newline.

Should be like 
 * @param string $mail
 *   The email address. The description can be longer if necessary, and if so,
 *   you can wrap it to another line.
 * @param string $from
 *   (optional) The email address to send the mail from, if different from
 *   the site-wide address.  

see https://www.drupal.org/coding-standards/docs#param